supabase/seed.sql

```diff
@@ -12,15 +12,64 @@
+                           ('00000000-0000-0000-0000-000000000000', 'd433e3e3-3e3e-3e3e-3e3e-3e3e3e3e3e3e', 'authenticated', 'authenticated', 'landlord1@example.com', '$2a$10$yZX3jy4aLusYmQ38XgXWT.qdEEwlUSGCHwqPF1PDEe7Xvj4ZbBKqu', '2024-02-23 22:20:45.041235+00', NULL        , ''                  , NULL                  , ''              , NULL              , ''                      , ''            , NULL                  , '2024-02-23 20:40:23.012312+00', '{"provider": "email", "providers": ["email"]}', '{}'                , NULL            , '2024-02-21 19:02:56.019983+00', '2024-02-21 19:02:56.027023+00', NULL   , NULL                , ''            , ''                  , NULL                  , ''                          , 0                            , NULL          , ''                      , NULL                      , false        , NULL        ),
+                           ('00000000-0000-0000-0000-000000000000', '002a708b-38cf-4e22-8814-2a1eadce35b2', 'authenticated', 'authenticated', 'test@gamil.com'      , '$2a$10$yZX3jy4aLusYmQ38XgXWT.qdEEwlUSGCHwqPF1PDEe7Xvj4ZbBKqu', '2024-02-23 22:20:45.041235+00', NULL        , ''                  , NULL                  , ''              , NULL              , ''                      , ''            , NULL                  , '2024-02-23 20:40:23.012312+00', '{"provider": "email", "providers": ["email"]}', '{}'                , NULL            , '2024-02-21 19:02:56.019983+00', '2024-02-21 19:02:56.027023+00', NULL   , NULL                , ''            , ''                  , NULL                  , ''                          , 0                            , NULL          , ''                      , NULL                      , false        , NULL        );
+
+-- Create Sample Properties
+--
+-- |                  ID                  |     Address   |
+-- | 1ececec8-4bbf-445f-8de0-f563caf0bf01 |  1 Test Road  |
+-- | 6a83d02b-9da1-4a4a-9719-05e8a8c9228d |  2 Test Road  |
+
+INSERT INTO "public"."properties" (                    "id"              ,   "address"  ) VALUES
+                                  ('1ececec8-4bbf-445f-8de0-f563caf0bf01', '1 Test Road'),
+                                  ('6a83d02b-9da1-4a4a-9719-05e8a8c9228d', '2 Test Road');
+
+
+-- Create Sample Landlord Private Profiles
+--
+-- |                 user_id              |   phone_number   |  postcode  |     country    | county | city | street |    house     |  first_name | last_name |
+-- | d3e3e3e3-3e3e-3e3e-3e3e-3e3e3e3e3e3e |     12345678     |  TES T12   | United Kingdom |  NULL  | NULL |  NULL  | 1 Test House |    May     |   Chris   |
+-- | d433e3e3-3e3e-3e3e-3e3e-3e3e3e3e3e3e |     87654321     |  TES T34   | United Kingdom |  NULL  | NULL |  NULL  | 2 Test House |    Joe     |    Doe    |
```

Comment:
I don't think we need this level of information duplication, and we are perhaps missing some information that cannot be as easily inferred. This comment would be more helpful if (instead of repeating the information that is right beneath it in plain text) it added listed (for example):

```
|    User    |   Owned properties   |                                  Used in Tests                                  |
| Landlord.1 |     1 Test Road      |  landlord-page-tests - Landlord has profile & Owned properties show in profile  |
```

As this lets anyone who comes in later understand who these example landlords are, and what they can add or change without breaking other tests. E.g. don't write a test to see if this landlord can claim a new property because that might interfere with a test to see whether there are no properties listed on  their profile